src/example-relay/src/graphql/locations/types/output/AddLocationError.js

```diff
@@ -0,0 +1,17 @@
+// @flow strict
+
+import { GraphQLObjectType, GraphQLString } from 'graphql';
+
+import ErrorInterface from '../../../types/interface/Error';
+
+export default new GraphQLObjectType({
+  name: 'AddLocationError',
```

Comment:
Maybe better to be more generic and call it just `InputError`?

Reply:
Yes, but my idea was to add some more specific fields later, something to tell that IATA code is incorrect or something. We have the `Error` interface which is the more generic.